packages/devtools_app/lib/src/charts/flutter/flame_chart.dart

```diff
@@ -739,3 +803,74 @@ mixin FlameChartColorMixin {
     _unknownColorOffset = 0;
   }
 }
+
+/// [ExtentDelegate] implementation for the case where size and position is
+/// known for each list item.
+class _ScrollingFlameChartRowExtentDelegate extends ExtentDelegate {
+  _ScrollingFlameChartRowExtentDelegate({
+    @required this.nodeRects,
+    @required this.zoom,
+    @required this.chartStartInset,
+    @required this.chartWidth,
+  }) {
+    recompute();
+  }
+
+  List<Rect> nodeRects = [];
```

Comment:
make these 4 members private

Reply:
they can you just can't use the `this.` syntactic sugar. in the constructor.
E.g.
```dart
class Foo {
  Foo({Bar bar}) : _bar = bar;
```

```